ui/src/components/Code/index.jsx

```diff
@@ -35,7 +35,10 @@ export default class Code extends Component {
 
   render() {
     const { children, language, className, ...props } = this.props;
-    const code = highlight(language, children, true).value;
+    const code = highlight(children, {
+      language,
```

Comment:
Today I learned about ECMAScript 2015's object property shorthand.

Reply:
Same,  for me aswell